code/modules/projectiles/guns/mounted.dm

```diff
@@ -210,6 +210,47 @@
 	undeploy_time = 0.5 SECONDS
 	max_integrity = 200
 
+
+//AGLS-37, or Automatic Grenade Launching System 37, a fully automatic mounted grenade launcher that fires fragmentation and HE shells, can't be turned.
+
+/obj/item/weapon/gun/standard_agls
+	name = "\improper AGLS-37 Kauser automatic grenade launcher"
+	desc = "The AGLS-37 automatic grenade launching IFF capable system, it's too heavy to be wielded or operated without the tripod. On the back, it reads: \"The Explosions and Fragmentation from this weapon ARE NOT friendly fire capable. Kauser is not obligated to buy you new body parts for you or your friends if you lose them.\"\nCan be deployed with Crtl-Click. It CANNOT be turned once deployed, due to a lack of swivels, pick it up to move your cone of fire. Can be repaired with a blowtorch once deployed."
+	w_class = WEIGHT_CLASS_HUGE
+	flags_equip_slot = ITEM_SLOT_BACK
+	icon = 'icons/Marine/marine-hmg.dmi'
+	icon_state = "ags"
+	fire_sound = 'sound/weapons/guns/fire/ags.ogg'
+	reload_sound = 'sound/weapons/guns/interact/minigun_cocked.ogg'
+	default_ammo_type = /obj/item/ammo_magazine/standard_agls
+	scatter = 0
+	fire_delay = 0.75 SECONDS
```

Comment:
Oh, lord. Lumi is right. 0.75 seconds? That's way faster than GL.
![image](https://user-images.githubusercontent.com/6610922/171313710-89894f3c-e06f-41ba-b14c-95093f5ca3fd.png)
![image](https://user-images.githubusercontent.com/6610922/171313766-1e7165b8-3e56-4846-adb6-0161e7db0f19.png)